sql/report_queries/erm/costs/erm_inventory_cost.sql

```diff
@@ -0,0 +1,179 @@
+        -- Please comment/uncomment one pair the these parameters if you want to define the date range of paid invoices
+        NULL::DATE AS start_date,
+        NULL::DATE AS end_date,
+        --'2000-01-01' :: DATE AS start_date, -- since there is no paymentDate now using approvalDate
+        --'2021-01-15' :: DATE AS end_date, -- since there is no paymentDate now using approvalDate
+        -- filters on invoice line level
+        ''::VARCHAR AS invoice_line_status, -- Enter your invoice line status eg. 'Paid', 'Approved', 'Open' etc.
+        -- filters on purchase order line level
+        ''::VARCHAR AS po_line_order_format, -- Enter your po line format eg. 'Electronic Resource', 'Physical Resource', 'P/E Mix' etc.
+        -- filters on instance level
+        ''::VARCHAR AS instance_subject, -- Enter your  eg.  etc.
+        ''::VARCHAR AS mode_of_issuance_name, -- Enter your mode_of_issuance_name eg.  etc.
+        'computer -- online resource'::VARCHAR AS format_name, -- Enter your format_name eg. 'computer -- online resource'  etc.
+        ''::VARCHAR AS library_name -- Enter your library_location_name eg.  etc.
+),
+     instance_detail AS (
+         SELECT
+             inst.instance_id AS "instance_id",
+             inst.mode_of_issuance_name AS "instance_mode_of_issuance_name",
+             inst_formats.format_name AS "instance_format_name",
+             inst_subjects.subject AS "instance_subject",
+             loc.library_name AS "instance_library_name"
+         FROM
+             folio_reporting.instance_ext AS inst
+                 LEFT JOIN folio_reporting.instance_formats AS inst_formats ON inst.instance_id = inst_formats.instance_id
+                 LEFT JOIN folio_reporting.instance_subjects AS inst_subjects ON inst.instance_id = inst_subjects.instance_id
+                 LEFT JOIN folio_reporting.holdings_ext AS hld ON inst.instance_id = hld.instance_id
+                 LEFT JOIN folio_reporting.locations_libraries AS loc ON hld.permanent_location_id = loc.location_id
+     ),
+-- bringing in no of subjects per instance for further calculations
+     instance_subs AS (
+         SELECT
+             subs.instance_id AS "instance_id",
+             count(DISTINCT subs.subject)::int AS "no_instance_subjects"
+         FROM
+             folio_reporting.instance_subjects AS subs
+         GROUP BY
+             subs.instance_id
+     ),
+-- bringing in no of formats per instance for further calculations
+     instance_formats AS (
+         SELECT
+             forms.instance_id AS "instance_id",
+             count(DISTINCT forms.format_name)::int AS "no_instance_formats"
+         FROM
+             folio_reporting.instance_formats AS forms
+         GROUP BY
+             forms.instance_id
+     ),
+     invoice_detail AS (
+         SELECT
+             inv.id AS "invoice_id",
+             inv.approval_date AS "invoice_approval_date"
+         FROM
+             invoice_invoices AS inv
```

Comment:
I ran this against snapshot and snapshot20210114_invoice hoping that there would be some data. In each case, there is no data. There is the empty column id but no data for approval_date. I commented out approval_date which removed the error but not that there is no data.

Reply:
Thank you for taking this PR to review and sorry for lacking some informations on how to set a proper test envrionment.
A good choice of testing this query is ` folio_snapshot_20210114_invoice `
But as it is using a new derived table you have to rename the joined `table folio_reporting.invoice_adjustments_ext` to `local.axeldoerrer_invoice_adjustments_ext` .
You then will receive only one row, which is unfortunately the only row which relies on electronic resources. But if you remove the preset filter value for format_name you'll see more rows which should give you an idea how that report is intended.